mail_debrand/data/mail_data.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     Serpent Consulting Services Pvt. Ltd.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0) -->
+<openerp>
+    <data>
```

Comment:
As stated, this shouldn't be changed in the inheritance chain. Although is interesting to modify the template by hand, this would break other possible inherited modules.